Formula/vgmstream.rb

```diff
@@ -1,11 +1,10 @@
 class Vgmstream < Formula
   desc "Library for playing streamed audio formats from video games"
   homepage "https://hcs64.com/vgmstream.html"
-  url "https://github.com/losnoco/vgmstream/archive/r1050-3280-gba405509.tar.gz"
-  version "r1050-3280-gba405509"
-  sha256 "ab5c900af26f642ffcf96e95eb83e79722e07751f861afe3cd8acf5239394f1d"
+  url "https://github.com/losnoco/vgmstream.git",
+      tag: "r1050-3280-gba405509"
```

Comment:
```suggestion
  url "https://github.com/losnoco/vgmstream.git",
      tag:      "r1050-3280-gba405509",
      revision: "xxxxxxxxxxx"
```

For git repos, we require specifying both a `tag` and a `revision` which points to the commit that it tag points to.